From 45405c0fc927c9ce0b20596db69b42561a16d95c Mon Sep 17 00:00:00 2001
From: David Gibson <dgibson@redhat.com>
Date: Mon, 15 Sep 2014 07:12:38 -0400
Subject: [virt] kvm/ppc: Get rid of KVM_HPAGE defines

Message-id: <1410765214-16377-23-git-send-email-dgibson@redhat.com>
Patchwork-id: 94753
O-Subject: [PATCH 22/78] KVM: PPC: Get rid of KVM_HPAGE defines
Bugzilla: 1123145 1123133 1123367
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1123145

Signed-off-by: David Gibson <dgibson@redhat.com>

commit 2c5350e934501f1af8010c608d8dbf72ad25fdc6
Author: Christoffer Dall <christoffer.dall@linaro.org>
Date:   Wed Oct 2 14:22:33 2013 -0700

    KVM: PPC: Get rid of KVM_HPAGE defines

    Now when the main kvm code relying on these defines has been moved to
    the x86 specific part of the world, we can get rid of these.

    Signed-off-by: Christoffer Dall <christoffer.dall@linaro.org>
    Signed-off-by: Gleb Natapov <gleb@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/include/asm/kvm_host.h b/arch/powerpc/include/asm/kvm_host.h
index 27196f7..84c24db 100644
--- a/arch/powerpc/include/asm/kvm_host.h
+++ b/arch/powerpc/include/asm/kvm_host.h
@@ -64,11 +64,6 @@ extern void kvm_set_spte_hva(struct kvm *kvm, unsigned long hva, pte_t pte);
 
 #endif
 
-/* We don't currently support large pages. */
-#define KVM_HPAGE_GFN_SHIFT(x) 0
-#define KVM_NR_PAGE_SIZES 1
-#define KVM_PAGES_PER_HPAGE(x) (1UL<<31)
-
 #define HPTEG_CACHE_NUM   (1 << 15)
 #define HPTEG_HASH_BITS_PTE  13
 #define HPTEG_HASH_BITS_PTE_LONG 12
-- 
1.7.1